airflow/gcp/hooks/pubsub.py

```diff
@@ -19,12 +19,18 @@
 """
 This module contains a Google Pub/Sub Hook.
 """
-from typing import Any, List, Dict, Optional
+from typing import List, Dict, Optional, Sequence, Tuple, Union
 from uuid import uuid4
 
-from googleapiclient.discovery import build
+from cached_property import cached_property
+from google.api_core.retry import Retry
+from google.api_core.exceptions import AlreadyExists, GoogleAPICallError
+from google.cloud.exceptions import NotFound
+from google.cloud.pubsub_v1 import PublisherClient, SubscriberClient
+from google.cloud.pubsub_v1.types import Duration, PushConfig, MessageStoragePolicy
 from googleapiclient.errors import HttpError
 
+from airflow.version import version
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
 
 
```

Comment:
Are the following methods no longer used? If they used, can you complete the tests? If they are not used, can you delete them?

Reply:
You are right. `_format_subscription` and `_format_topic` are not longer needed. I will delete them.